src/languages/sqf.js

```diff
@@ -82,351 +83,394 @@ export default function(hljs) {
+        'textlogformat tg time timemultiplier titlecut ' +
+        'titleobj titlersc titletext toarray tofixed ' +
+        'toloweransi tostring toupper toupperansi triggeractivated ' +
+        'triggerammo triggerarea triggerattachedvehicle triggerattachobject triggerattachvehicle ' +
+        'triggerinterval triggerstatements triggertext triggertimeout triggertimeoutcurrent ' +
+        'trim turretlocal turretowner turretunit tvadd ' +
+        'tvcollapse tvcollapseall tvcount tvcursel tvdata ' +
+        'tvexpand tvexpandall tvisselected tvpicture tvpictureright ' +
+        'tvsetcolor tvsetcursel tvsetdata tvsetpicture tvsetpicturecolor ' +
+        'tvsetpicturecolorselected tvsetpictureright tvsetpicturerightcolor tvsetpicturerightcolordisabled tvsetpicturerightcolorselected ' +
+        'tvsetselected tvsettext tvsettooltip tvsetvalue tvsort ' +
+        'tvsortbyvalue tvsortbyvalueall tvtext tvtooltip tvvalue ' +
+        'typename typeof uavcontrol uinamespace uisleep ' +
+        'unassignitem unassignteam unassignvehicle underwater uniform ' +
+        'uniformitems uniformmagazines unitaddons unitaimposition unitaimpositionvisual ' +
+        'unitcombatmode unitisuav unitpos unitready unitrecoilcoefficient ' +
+        'unitsbelowheight unitturret unlinkitem unlockachievement unregistertask ' +
+        'updatemenuitem updateobjecttree useaiopermapobstructiontest useaisteeringcomponent useaudiotimeformoves ' +
+        'vectoradd vectorcos vectorcrossproduct vectordiff vectordir ' +
+        'vectordistance vectordistancesqr vectordotproduct vectorfromto vectorlinearconversion ' +
+        'vectormagnitudesqr vectormodeltoworld vectormodeltoworldvisual vectormultiply vectornormalized ' +
+        'vectorupvisual vectorworldtomodel vectorworldtomodelvisual vehicle vehiclecargoenabled ' +
+        'vehiclemoveinfo vehicleradio vehiclereceiveremotetargets vehiclereportownposition vehiclereportremotetargets ' +
+        'vehiclevarname velocity velocitymodelspace verifysignature vest ' +
+        'vestitems vestmagazines viewdistance visiblecompass visiblegps ' +
+        'visibleposition visiblepositionasl visiblescoretable visiblewatch waituntil ' +
+        'waypointattachedobject waypointattachedvehicle waypointattachobject waypointattachvehicle waypointbehaviour ' +
+        'waypointcompletionradius waypointdescription waypointforcebehaviour waypointformation waypointhouseposition ' +
+        'waypointloiterradius waypointloitertype waypointname waypointposition waypoints ' +
+        'waypointsenableduav waypointshow waypointspeed waypointstatements waypointtimeout ' +
+        'waypointtype waypointvisible weaponaccessories weaponaccessoriescargo weaponcargo ' +
+        'weaponinertia weaponlowered weapons weaponsitems weaponsitemscargo ' +
+        'weaponsturret weightrtd wfsidetext wind winddir ' +
+        'windstr wingsforcesrtd worldname worldsize worldtomodel worldtoscreen ',
```

Comment:
This will also need to be converted into an array now to make maintenance (and review) in the future easier. See `ecmascript.js` for examples.

Reply:
Thanks, will do.